_posts/2025-02-01-next-scala-lts.md

```diff
@@ -0,0 +1,117 @@
+---
+category: announcement
+permalink: /news/next-scala-lts.html
+title: "Next Scala 3 LTS version"
+by: Tomasz Godzik
+---
+
+**TLDR;**
+
+API used in lazy val implementation, will be removed in a future JDK thus one of
+the Scala 3 Next minors in 2025 Q4 and the next LTS will drop JDK 8 support.
```

Comment:
This should not be enough trigger I think, If we switch to `AtomicReferenceFieldUpdater`, then problem is gone.

Reply:
`AtomicLongFieldUpdater` is the right tool. The actual field holding the computed value is accessed plain; it's safe due to the release-acquire effects of the corresponding long bit map field, that it's written before bits are written, and read after the bits are read.

Reply:
We do try to list more reasons than just it. TLDR is just short version of that. And we can always discuss everything properly under the contributors thread